File: src/main/java/me/lokka30/levelledmobs/result/PlayerNetherOrWorldSpawnResult.java
package me.lokka30.levelledmobs.result;

import org.bukkit.Location;
import org.jetbrains.annotations.NotNull;

/**
 * Used to hold info that is used for various
 * custom placeholders
 *
 * @author stumper66
 * @since 3.6.0
 */
public record PlayerNetherOrWorldSpawnResult(@NotNull Location location, boolean isNetherPortalLocation,
                                             boolean isWorldPortalLocation) {

}

File: levelledmobs-plugin/src/main/kotlin/io/github/arcaneplugins/levelledmobs/result/PlayerNetherOrWorldSpawnResult.kt
package io.github.arcaneplugins.levelledmobs.result

import org.bukkit.Location

/**
 * Used to hold info that is used for various
 * custom placeholders
 *
 * @author stumper66
 * @since 3.6.0
 */
class PlayerNetherOrWorldSpawnResult(
    val location: Location?,
    var isNetherPortalLocation: Boolean,
    var isWorldPortalLocation: Boolean
)